ko/sql/schema/table_stmt.rst

```diff
@@ -1614,15 +1614,21 @@ CHANGE/MODIFY 절
 
 *   **NOT NULL**
 
-    *   변경할 칼럼에 **NOT NULL** 제약 조건이 지정되지 않으면 기존 테이블에 존재하더라도 새 테이블에서 제거된다.
-    *   변경할 칼럼에 **NOT NULL** 제약 조건이 지정되면 시스템 파라미터 **alter_table_change_type_strict** 의 설정에 따라 결과가 달라진다.
+    *   변경할 칼럼에 **NOT NULL** 속성이 지정되지 않더라도 새 테이블에서 해당 속성이 제거되지 않고 그대로 남아있게 된다. 새 테이블에서 해당 속성을 제거하려면 구문에서 ** NULL **\을 지정하면 된다.
```

Comment:
Please remove spaces before and after "NULL".
```
** NULL **\을
```
to
```
**NULL**\을
```

Reply:
I fixed it, thanks.